libs/tunables/tunables_shell.py

```diff
@@ -0,0 +1,113 @@
+import cmd
+import yaml
+
+from ast import literal_eval
+from datetime import date
+
+
+# import database
+
+
+class tunables(cmd.Cmd):
+    print(
+        " \n This is the Tunables Parameters program. \n Type in ? to see the list of commands"
+    )
+    prompt = "(tunables) "
+    file = None
```

Comment:
Based on how you're parsing the user's input, it seems that you are expecting a certain format for the data to come in, might be helpful to prompt the user about what format you are expecting just to avoid misuse or incorrectly passed in parameters